### ISSUE TITLE
2025-03-31, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#126]
# Today's Arxiv Papers

(1) [Q-Insight: Understanding Image Quality via Visual Reinforcement Learning](http://arxiv.org/abs/2503.22679v1), 2025-03-28

(2) [Self-Evolving Multi-Agent Simulations for Realistic Clinical Interactions](http://arxiv.org/abs/2503.22678v1), 2025-03-28

(3) [QuestBench: Can LLMs ask the right question to acquire information in reasoning tasks?](http://arxiv.org/abs/2503.22674v1), 2025-03-28

(4) [Exploring the Effectiveness of Multi-stage Fine-tuning for Cross-encoder Re-rankers](http://arxiv.org/abs/2503.22672v1), 2025-03-28

(5) [Unicorn: Text-Only Data Synthesis for Vision Language Model Training](http://arxiv.org/abs/2503.22655v1), 2025-03-28

(6) [Evaluating Multimodal Language Models as Visual Assistants for Visually Impaired Users](http://arxiv.org/abs/2503.22610v1), 2025-03-28

(7) [On the Alignment of Post-Publication Reviews & Bibliometric and Altmetric Impact -- A Case Study on Expert Statements from the Science Media Center Germany](http://arxiv.org/abs/2503.22594v1), 2025-03-28

(8) [Using AI to Summarize US Presidential Campaign TV Advertisement Videos, 1952-2012](http://arxiv.org/abs/2503.22589v1), 2025-03-28

(9) [LLM-enabled Instance Model Generation](http://arxiv.org/abs/2503.22587v1), 2025-03-28

(10) [Historical Ink: Exploring Large Language Models for Irony Detection in 19th-Century Spanish](http://arxiv.org/abs/2503.22585v1), 2025-03-28

(11) [Beyond Vanilla Fine-Tuning: Leveraging Multistage, Multilingual, and Domain-Specific Methods for Low-Resource Machine Translation](http://arxiv.org/abs/2503.22582v1), 2025-03-28

(12) [Drop the Golden Apples: Identifying Third-Party Reuse by DB-Less Software Composition Analysis](http://arxiv.org/abs/2503.22576v1), 2025-03-28

(13) [Niyama : Breaking the Silos of LLM Inference Serving](http://arxiv.org/abs/2503.22562v1), 2025-03-28

(14) [Bridging the Dimensional Chasm: Uncover Layer-wise Dimensional Reduction in Transformers through Token Correlation](http://arxiv.org/abs/2503.22547v1), 2025-03-28

(15) [Exploiting Mixture-of-Experts Redundancy Unlocks Multimodal Generative Abilities](http://arxiv.org/abs/2503.22517v1), 2025-03-28

(16) [Unlocking LLM Repair Capabilities in Low-Resource Programming Languages Through Cross-Language Translation and Multi-Agent Refinement](http://arxiv.org/abs/2503.22512v1), 2025-03-28

(17) [Probabilistic Uncertain Reward Model: A Natural Generalization of Bradley-Terry Reward Model](http://arxiv.org/abs/2503.22480v1), 2025-03-28

(18) [WorkTeam: Constructing Workflows from Natural Language with Multi-Agents](http://arxiv.org/abs/2503.22473v1), 2025-03-28

(19) [Evaluating LLM-based Agents for Multi-Turn Conversations: A Survey](http://arxiv.org/abs/2503.22458v1), 2025-03-28

(20) [Entropy-guided sequence weighting for efficient exploration in RL-based LLM fine-tuning](http://arxiv.org/abs/2503.22456v1), 2025-03-28

(21) [STADE: Standard Deviation as a Pruning Metric](http://arxiv.org/abs/2503.22451v1), 2025-03-28

(22) [NuGrounding: A Multi-View 3D Visual Grounding Framework in Autonomous Driving](http://arxiv.org/abs/2503.22436v1), 2025-03-28

(23) [CoSIL: Software Issue Localization via LLM-Driven Code Repository Graph Searching](http://arxiv.org/abs/2503.22424v1), 2025-03-28

(24) [Unveiling the Mist over 3D Vision-Language Understanding: Object-centric Evaluation with Chain-of-Analysis](http://arxiv.org/abs/2503.22420v1), 2025-03-28

(25) [Training Large Language Models for Advanced Typosquatting Detection](http://arxiv.org/abs/2503.22406v1), 2025-03-28

(26) [EllieSQL: Cost-Efficient Text-to-SQL with Complexity-Aware Routing](http://arxiv.org/abs/2503.22402v1), 2025-03-28

(27) [Generative Reliability-Based Design Optimization Using In-Context Learning Capabilities of Large Language Models](http://arxiv.org/abs/2503.22401v1), 2025-03-28

(28) [Negation: A Pink Elephant in the Large Language Models' Room?](http://arxiv.org/abs/2503.22395v1), 2025-03-28

(29) [Why Stop at One Error? Benchmarking LLMs as Data Science Code Debuggers for Multi-Hop and Multi-Bug Errors](http://arxiv.org/abs/2503.22388v1), 2025-03-28

(30) [Supposedly Equivalent Facts That Aren't? Entity Frequency in Pre-training Induces Asymmetry in LLMs](http://arxiv.org/abs/2503.22362v1), 2025-03-28

(31) [Firm or Fickle? Evaluating Large Language Models Consistency in Sequential Interactions](http://arxiv.org/abs/2503.22353v1), 2025-03-28

(32) [Using a Large Language Model as Design Material for an Interactive Museum Installation](http://arxiv.org/abs/2503.22345v1), 2025-03-28

(33) [SKDU at De-Factify 4.0: Natural Language Features for AI-Generated Text-Detection](http://arxiv.org/abs/2503.22338v1), 2025-03-28

(34) [A Refined Analysis of Massive Activations in LLMs](http://arxiv.org/abs/2503.22329v1), 2025-03-28

(35) [Large Language Models Are Democracy Coders with Attitudes](http://arxiv.org/abs/2503.22315v1), 2025-03-28

(36) [Preference-based Learning with Retrieval Augmented Generation for Conversational Question Answering](http://dx.doi.org/10.1145/3701716.3715544), 2025-03-28

(37) [BanglAssist: A Bengali-English Generative AI Chatbot for Code-Switching and Dialect-Handling in Customer Service](http://arxiv.org/abs/2503.22283v1), 2025-03-28

(38) [MultiClaimNet: A Massively Multilingual Dataset of Fact-Checked Claim Clusters](http://arxiv.org/abs/2503.22280v1), 2025-03-28

(39) [Make Some Noise: Towards LLM audio reasoning and generation using sound tokens](http://dx.doi.org/10.1109/ICASSP49660.2025.10888809), 2025-03-28

(40) [Beyond the Script: Testing LLMs for Authentic Patient Communication Styles in Healthcare](http://arxiv.org/abs/2503.22250v1), 2025-03-28

(41) [Agent-Centric Personalized Multiple Clustering with Multi-Modal LLMs](http://arxiv.org/abs/2503.22241v1), 2025-03-28

(42) [Integrating LLMs in Software Engineering Education: Motivators, Demotivators, and a Roadmap Towards a Framework for Finnish Higher Education Institutes](http://arxiv.org/abs/2503.22238v1), 2025-03-28

(43) [Exploring Data Scaling Trends and Effects in Reinforcement Learning from Human Feedback](http://arxiv.org/abs/2503.22230v1), 2025-03-28

(44) [Learning to Instruct for Visual Instruction Tuning](http://arxiv.org/abs/2503.22215v1), 2025-03-28

(45) [DeepSound-V1: Start to Think Step-by-Step in the Audio Generation from Videos](http://arxiv.org/abs/2503.22208v1), 2025-03-28

(46) [EdgeInfinite: A Memory-Efficient Infinite-Context Transformer for Edge Devices](http://arxiv.org/abs/2503.22196v1), 2025-03-28

(47) [Reasoning of Large Language Models over Knowledge Graphs with Super-Relations](http://arxiv.org/abs/2503.22166v1), 2025-03-28

(48) [Landscape of Thoughts: Visualizing the Reasoning Process of Large Language Models](http://arxiv.org/abs/2503.22165v1), 2025-03-28

(49) [PharmAgents: Building a Virtual Pharma with Large Language Model Agents](http://arxiv.org/abs/2503.22164v1), 2025-03-28

(50) [Traffic Modeling for Network Security and Privacy: Challenges Ahead](http://arxiv.org/abs/2503.22161v1), 2025-03-28

(51) [EgoToM: Benchmarking Theory of Mind Reasoning from Egocentric Videos](http://arxiv.org/abs/2503.22152v1), 2025-03-28

(52) [Tokenization of Gaze Data](http://arxiv.org/abs/2503.22145v1), 2025-03-28

(53) [FRASE: Structured Representations for Generalizable SPARQL Query Generation](http://arxiv.org/abs/2503.22144v1), 2025-03-28

(54) [Sharpe Ratio-Guided Active Learning for Preference Optimization in RLHF](http://arxiv.org/abs/2503.22137v1), 2025-03-28

(55) [Beyond Single-Sentence Prompts: Upgrading Value Alignment Benchmarks with Dialogues and Stories](http://arxiv.org/abs/2503.22115v1), 2025-03-28

(56) [M2D2: Exploring General-purpose Audio-Language Representations Beyond CLAP](http://arxiv.org/abs/2503.22104v1), 2025-03-28

(57) [Few-Shot Graph Out-of-Distribution Detection with LLMs](http://arxiv.org/abs/2503.22097v1), 2025-03-28

(58) [Leveraging LLMs for Predicting Unknown Diagnoses from Clinical Notes](http://arxiv.org/abs/2503.22092v1), 2025-03-28

(59) [Penrose Tiled Low-Rank Compression and Section-Wise Q&A Fine-Tuning: A General Framework for Domain-Specific Large Language Model Adaptation](http://arxiv.org/abs/2503.22074v1), 2025-03-28

(60) [Arch-LLM: Taming LLMs for Neural Architecture Generation via Unsupervised Discrete Representation Learning](http://arxiv.org/abs/2503.22063v1), 2025-03-28

(61) [ThinkEdit: Interpretable Weight Editing to Mitigate Overly Short Thinking in Reasoning Models](http://arxiv.org/abs/2503.22048v1), 2025-03-27

(62) [The Risks of Using Large Language Models for Text Annotation in Social Science Research](http://arxiv.org/abs/2503.22040v1), 2025-03-27

(63) [Debate-Driven Multi-Agent LLMs for Phishing Email Detection](http://arxiv.org/abs/2503.22038v1), 2025-03-27

(64) [Cognitive Prompts Using Guilford's Structure of Intellect Model](http://arxiv.org/abs/2503.22036v1), 2025-03-27

(65) [Enhancing Domain-Specific Encoder Models with LLM-Generated Data: How to Leverage Ontologies, and How to Do Without Them](http://arxiv.org/abs/2503.22006v1), 2025-03-27

(66) [Empowering Retrieval-based Conversational Recommendation with Contrasting User Preferences](http://arxiv.org/abs/2503.22005v1), 2025-03-27

(67) [Socially Constructed Treatment Plans: Analyzing Online Peer Interactions to Understand How Patients Navigate Complex Medical Conditions](http://arxiv.org/abs/2503.21986v1), 2025-03-27

(68) [Learning to Lie: Reinforcement Learning Attacks Damage Human-AI Teams and Teams of LLMs](http://arxiv.org/abs/2503.21983v1), 2025-03-27

(69) [RocketPPA: Ultra-Fast LLM-Based PPA Estimator at Code-Level Abstraction](http://arxiv.org/abs/2503.21971v1), 2025-03-27

(70) [Data-Agnostic Robotic Long-Horizon Manipulation with Vision-Language-Guided Closed-Loop Feedback](http://arxiv.org/abs/2503.21969v1), 2025-03-27

(71) [Entropy-Aware Branching for Improved Mathematical Reasoning](http://arxiv.org/abs/2503.21961v1), 2025-03-27

(72) [Proof or Bluff? Evaluating LLMs on 2025 USA Math Olympiad](http://arxiv.org/abs/2503.21934v1), 2025-03-27

(73) [Local Normalization Distortion and the Thermodynamic Formalism of Decoding Strategies for Large Language Models](http://arxiv.org/abs/2503.21929v1), 2025-03-27

(74) [Hybrid Emotion Recognition: Enhancing Customer Interactions Through Acoustic and Textual Analysis](http://arxiv.org/abs/2503.21927v1), 2025-03-27

(75) [AutoPsyC: Automatic Recognition of Psychodynamic Conflicts from Semi-structured Interviews with Large Language Models](http://arxiv.org/abs/2503.21911v1), 2025-03-27

(76) [AssistPDA: An Online Video Surveillance Assistant for Video Anomaly Prediction, Detection, and Analysis](http://arxiv.org/abs/2503.21904v1), 2025-03-27

(77) [OntoAligner: A Comprehensive Modular and Robust Python Toolkit for Ontology Alignment](http://arxiv.org/abs/2503.21902v1), 2025-03-27

(78) [RedditESS: A Mental Health Social Support Interaction Dataset -- Understanding Effective Social Support to Refine AI-Driven Support Tools](http://arxiv.org/abs/2503.21888v1), 2025-03-27

(79) [Video-R1: Reinforcing Video Reasoning in MLLMs](http://arxiv.org/abs/2503.21776v1), 2025-03-27

(80) [MemInsight: Autonomous Memory Augmentation for LLM Agents](http://arxiv.org/abs/2503.21760v1), 2025-03-27

(81) [Fwd2Bot: LVLM Visual Token Compression with Double Forward Bottleneck](http://arxiv.org/abs/2503.21757v1), 2025-03-27

(82) [VBench-2.0: Advancing Video Generation Benchmark Suite for Intrinsic Faithfulness](http://arxiv.org/abs/2503.21755v1), 2025-03-27

(83) [GateLens: A Reasoning-Enhanced LLM Agent for Automotive Software Release Analytics](http://arxiv.org/abs/2503.21735v1), 2025-03-27

(84) [Effective Skill Unlearning through Intervention and Abstention](http://arxiv.org/abs/2503.21730v1), 2025-03-27

(85) [Collab: Controlled Decoding using Mixture of Agents for LLM Alignment](http://arxiv.org/abs/2503.21720v1), 2025-03-27

(86) [CLAIMCHECK: How Grounded are LLM Critiques of Scientific Papers?](http://arxiv.org/abs/2503.21717v1), 2025-03-27

(87) [Enhancing Repository-Level Software Repair via Repository-Aware Knowledge Graphs](http://arxiv.org/abs/2503.21710v1), 2025-03-27

(88) [LLM-Gomoku: A Large Language Model-Based System for Strategic Gomoku with Self-Play and Reinforcement Learning](http://arxiv.org/abs/2503.21683v1), 2025-03-27

(89) [JiraiBench: A Bilingual Benchmark for Evaluating Large Language Models' Detection of Human Self-Destructive Behavior Content in Jirai Community](http://arxiv.org/abs/2503.21679v1), 2025-03-27

(90) [How do language models learn facts? Dynamics, curricula and hallucinations](http://arxiv.org/abs/2503.21676v1), 2025-03-27

(91) [Intelligent IoT Attack Detection Design via ODLLM with Feature Ranking-based Knowledge Base](http://arxiv.org/abs/2503.21674v1), 2025-03-27

(92) [COMI-LINGUA: Expert Annotated Large-Scale Dataset for Multitask NLP in Hindi-English Code-Mixing](http://arxiv.org/abs/2503.21670v1), 2025-03-27

(93) [UI-R1: Enhancing Action Prediction of GUI Agents by Reinforcement Learning](http://arxiv.org/abs/2503.21620v1), 2025-03-27

(94) [A Measure Based Generalizable Approach to Understandability](http://arxiv.org/abs/2503.21615v1), 2025-03-27

(95) [Evaluating book summaries from internal knowledge in Large Language Models: a cross-model and semantic consistency approach](http://arxiv.org/abs/2503.21613v1), 2025-03-27

(96) [GenEdit: Compounding Operators and Continuous Improvement to Tackle Text-to-SQL in the Enterprise](http://arxiv.org/abs/2503.21602v1), 2025-03-27

(97) [Prompt, Divide, and Conquer: Bypassing Large Language Model Safety Filters via Segmented and Distributed Prompt Processing](http://arxiv.org/abs/2503.21598v1), 2025-03-27

(98) [Cooking Task Planning using LLM and Verified by Graph Network](http://arxiv.org/abs/2503.21564v1), 2025-03-27

(99) [debug-gym: A Text-Based Environment for Interactive Debugging](http://arxiv.org/abs/2503.21557v1), 2025-03-27

(100) [SWI: Speaking with Intent in Large Language Models](http://arxiv.org/abs/2503.21544v1), 2025-03-27
